26-register_registration/resource_types/research_data/creator_selects_resource_type_research_data_and_subtype_dataset.feature

```diff
@@ -0,0 +1,90 @@
+Feature: Creator selects Resource type Research Data and Subtype Dataset
+
+    Scenario: Creator sees fields for Dataset subtype
+        Given Creator navigates to the Resource Type tab and selects Resource type "Research Data"
+        When Creator select the Resource Subtype "Dataset"
+        And Creator see a question about personal information and confidential data
+        And two buttons
+            | Yes |
+            | No  |
+
+    Scenario: Creator registers a confidential Dataset
+        Given Creator sees fields for Dataset subtype
+        When Creator select "Yes"
+        Then Creator see a user support dialog and is advised to seek help
+
+    Scenario: Creator registers a Dataset
+        Given Creator sees fields for Dataset subtype
+        When Creator select "No"
+        Then Creator see fields:
+            | Publisher                 |
+            | Process/method            |
+            | Country                   |
+            | GPS-coordinates           |
+            | Geographic area           |
+            | Data Management Plan      |
+            | Publication using Dataset |
+            | Other related resources   |
```

Comment:
I skisser er det et felt "Lenke til data" helt øverst. Skal den inn her? Er det samme som DOI-lenke for alle andre typer?

Reply:
I scenario under her nevnes et felt "Link to original version". Er det det samme?